config.json

```diff
@@ -258,7 +258,7 @@
         "prerequisites": [],
```

Comment:
All the "topics" fields below should not be changed. Topics are not used anymore by the website but they serve as orientation when setting "prerequisites" and "practices" keys. The "topic" of those exercises is still "strings", not "string-formatting".